sdk/include/platform/sunburst/platform-pwm.hh

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <debug.hh>
+#include <stdint.h>
+
+/**
+ * A driver for Sonata's PWM.
+ *
+ * Documentation source can be found at:
+ * https://github.com/lowRISC/sonata-system/blob/97a525c48f7bf051b999d0178dba04859819bc5e/doc/ip/pwm.md
+ *
+ * Rendered documentation is served from:
+ * https://lowrisc.github.io/sonata-system/doc/ip/pwm.html
+ */
+struct SonataPwm
+{
+	/**
+	 * Flag to set when debugging the driver for UART log messages.
+	 */
+	static constexpr bool DebugPwm = false;
+
+	/**
+	 * Helper for conditional debug logs and assertions.
+	 */
+	using Debug = ConditionalDebug<DebugPwm, "PWM">;
+
+	/**
+	 * The number of PWMs that are initialised.
```

Comment:
```suggestion
	 * The number of PWM outputs available.
```

Reply:
Agreed - also changing the name `pwmCount` to `pwmOutputCount` so this is more explicit that we are talking about possible PWM outputs and not individual devices.